src/modules/did/did.service.ts

```diff
@@ -486,7 +511,7 @@ export class DIDService implements OnModuleInit, OnModuleDestroy {
 
   private async pinDocument(did: string): Promise<void> {
     try {
-      await this.didQueue.add(UPDATE_DID_DOC_JOB_NAME, did);
+      await this.didQueue.add(UPDATE_DID_DOC_JOB_NAME, { did });
```

Comment:
@JGiter does this change relate to this comment you made in the PR description?
> Fixed using of DID synchronization job

Does the data need to be in an object?

Reply:
I can not say why, but when passing job as a string, I observed multiple errors `error parsing JSON` though there are no seemingly JSON parsing in DidProcessor. So I decided to make job as an object, same as in PinProcessor.